integration-tests/contracts/ethereum/src/KeeperBase.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+
+contract KeeperBase {
```

Comment:
this contract already exists in core... would it be better to just reuse the existing ones?

Reply:
Contracts that are used for tests like KeeperConsumer.sol don't exist in core. Only those contracts are kept in here. Replaced KeeperBase and KeeperCompaitableInterface with symlinks.